src/axe-reporter-earl/__tests__/axeReporterEarl.ts

```diff
@@ -0,0 +1,120 @@
+import axeReporterEarl, { earlUntested, concatReport } from "../axeReporterEarl";
+import context from '../context';
+import raw from '../../__test-utils__/data/raw-results-73f2c2.json';
+import { Env, RawResult } from "../../types";
+import 'jest-extended';
+
+const rawResults = raw as unknown as RawResult[][]
+
+describe(`axe-reporter-earl`, () => {
+  describe(`axeReporterEarl fn`, () => {
+    test(`get report with empty "@graph"`, () => {
+      const env = { url: '', version: '1.1.1' }
+      const actual = axeReporterEarl([], env);
+      expect(actual).toContainAllKeys(["@context", "@graph"]);
+      
+      expect(actual["@context"]).toBeObject();
+      expect(
+        JSON.stringify(actual["@context"]) === JSON.stringify(context)
```

Comment:
Any particular reason to stringify and compare rather than using `deepEquals(actual["@context"], context)`?

Reply:
It's to deal with differences of undefined.